### PR TITLE
ihor.bohdan

### PR DESCRIPTION
**Pull request by noxvail**

### CODE REVIEW

File: ihor.bohdan/second/task 1/tamagochi.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  end
+
+  def sickness_check
+    @health -= 30 if rand(1..100) > 90
+  end
+end
+
+pet = Tamagochi.new
+
+loop do
```
Comment (slenderock) [inline ihor.bohdan/second/task 1/tamagochi.rb]: lets have it as a part of pet methods maybe? what do you think?

Comment (noxvail) [reply ihor.bohdan/second/task 1/tamagochi.rb]: Done

File: ihor.bohdan/second/task 1/tamagochi.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
+class Tamagochi
+  attr_accessor :health
```
Comment (Flippylolz) [inline ihor.bohdan/second/task 1/tamagochi.rb]: As you decided to declare attr_accessor you need to actually use it. For example line 8 should be `self.health = health`

File: ihor.bohdan/second/task 1/tamagochi.rb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    @hunger = hunger
+    @thirst = thirst
+  end
+
+  def start
+    loop do
+      puts 'What do you want to do with your pet?'
+      action = gets.chomp.downcase
+      NAMES.include?(action) ? send(action) : start_error
+      break if @health.eql? 0
```
Comment (Flippylolz) [inline ihor.bohdan/second/task 1/tamagochi.rb:16]: Plase use attr_accesor

File: ihor.bohdan/second/task 1/tamagochi.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
+class Tamagochi
+  attr_accessor :health
+
+  NAMES = %w[rules state play feed hydrate].freeze
```
Comment (Flippylolz) [inline ihor.bohdan/second/task 1/tamagochi.rb]: name of constant should describe what's inside, maybe we should rename it? Thoughts?

**Review by Flippylolz — APPROVED** [review]
As there are no major issues here, I'll give you some time to apply fixes, if you want. This request will be merged soon.

File: ihor.bohdan/second/task 1/tamagochi.rb (hunk 2 above)
Comment (noxvail) [reply ihor.bohdan/second/task 1/tamagochi.rb]: done

File: ihor.bohdan/second/task 1/tamagochi.rb (hunk 3 above)
Comment (noxvail) [reply ihor.bohdan/second/task 1/tamagochi.rb:16]: done

File: ihor.bohdan/second/task 1/tamagochi.rb (hunk 4 above)
Comment (noxvail) [reply ihor.bohdan/second/task 1/tamagochi.rb]: done

**Pull request merged**